targets/f7/furi_hal/furi_hal_usb_ccid.c

```diff
@@ -364,10 +364,11 @@ void CALLBACK_CCID_XfrBlock(
             if(callbacks[CCID_SLOT_INDEX] != NULL) {
                 callbacks[CCID_SLOT_INDEX]->xfr_datablock_callback(
                     (const uint8_t*)receivedXfrBlock->abData,
-                    receivedXfrBlock->dwLength,
+                    (uint8_t)receivedXfrBlock->dwLength,
                     responseDataBlock->abData,
-                    &responseDataBlock->dwLength,
+                    (uint8_t*)&responseDataBlock->dwLength,
```

Comment:
and here. 

I know that you checking sizes in EP callback handling code, but still.